modules/oe_list_pages_open_vocabularies/tests/src/Kernel/ListPagesSearchApiConfiguratorTest.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_list_pages_open_vocabularies\Kernel;
+
+use Drupal\facets\Entity\Facet;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\KernelTests\Core\Entity\EntityKernelTestBase;
+use Drupal\open_vocabularies\Entity\OpenVocabulary;
+use Drupal\open_vocabularies\Entity\OpenVocabularyAssociation;
+use Drupal\search_api\Entity\Index;
+
+/**
+ * Tests the list page open vocabularies configurator.
+ */
+class ListPagesSearchApiConfiguratorTest extends EntityKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'system',
+    'field',
+    'user',
+    'datetime',
+    'datetime_range',
+    'emr',
+    'entity_reference_revisions',
+    'emr_node',
+    'link',
+    'node',
+    'oe_list_pages',
+    'oe_list_pages_filters_test',
+    'oe_list_page_content_type',
+    'oe_list_pages_open_vocabularies',
+    'open_vocabularies',
+    'open_vocabularies_test',
+    'options',
+    'search_api',
+    'search_api_db',
+    'search_api_test_db',
+    'taxonomy',
+    'text',
+    'facets',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    $this->installEntitySchema('search_api_task');
+    $this->installSchema('search_api', ['search_api_item']);
+    $this->installEntitySchema('facets_facet');
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('open_vocabulary');
+    $this->installEntitySchema('open_vocabulary_association');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('taxonomy_term');
+    $this->installEntitySchema('entity_meta');
+    $this->installEntitySchema('entity_meta_relation');
+    $this->installConfig(['node',
+      'open_vocabularies',
+      'oe_list_pages',
+      'oe_list_page_content_type',
+      'oe_list_pages_filters_test',
+      'emr',
+      'emr_node',
+    ]);
+
+    $storage = FieldStorageConfig::create([
+      'field_name' => 'open_vocs',
+      'entity_type' => 'node',
+      'type' => 'open_vocabulary_reference',
+    ]);
+    $storage->save();
+
+    // Add the OpenVocabularies fields to the content types.
+    foreach (['content_type_one', 'content_type_two'] as $bundle) {
+
+      $instance = FieldConfig::create([
```

Comment:
SInce we are probably gonna test all sorts of other things also with open vocabularies, shouldnt we ship with some fields already in the test module?